### PR TITLE
NOISSUE - Add delete certs feature

### PR DESCRIPTION
**Pull request by nyagamunene**
# What type of PR is this?
This is a feature because it adds the following functionality: it adds a feature for deleting multiple certs for an entity id. 

## What does this do?
It adds a feature for deleting multiple certs for an entity id. 

## Have you included tests for your changes?
Yes

## Did you document any new/modified features?
No

### CODE REVIEW

**Review by SammyOina — APPROVED** [review]

File: api/http/requests.go [hunk 1]
```diff
@@ -32,6 +32,17 @@ func (req viewReq) validate() error {
 	return nil
 }
 
+type deleteReq struct {
+	entityID string
```
Comment (dborovcanin) [inline api/http/requests.go:36]: Use `certID` or just `id`.

File: certs.go [hunk 2]
```diff
@@ -90,4 +93,7 @@ type Repository interface {
 
 	// ListRevokedCerts retrieves revoked lists from database.
 	ListRevokedCerts(ctx context.Context) ([]Certificate, error)
+
+	// RemoveCerts deletes certs from database.
+	RemoveCerts(ctx context.Context, entityId string) error
```
Comment (dborovcanin) [inline certs.go:98]: Use singular `RemovCert`.

Comment (dborovcanin) [reply certs.go:98]: Also `certID` or `id`.

File: sdk/sdk.go [hunk 3]
```diff
@@ -291,6 +298,12 @@ func (sdk mgSDK) ListCerts(pm PageMetadata) (CertificatePage, errors.SDKError) {
 	return cp, nil
 }
 
+func (sdk mgSDK) DeleteCerts(entityID string) errors.SDKError {
```
Comment (dborovcanin) [inline sdk/sdk.go:301]: Same, use singular `DeleteCert`.

File: service.go [hunk 4]
```diff
@@ -271,6 +271,10 @@ func (s *service) ListCerts(ctx context.Context, pm PageMetadata) (CertificatePa
 	return certPg, nil
 }
 
+func (s *service) RemoveCerts(ctx context.Context, entityId string) error {
```
Comment (dborovcanin) [inline service.go:274]: Same, use singular.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**